oneview/data_source_server_certificate.go

```diff
@@ -0,0 +1,368 @@
+						},
+						"issuer": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"key_usage": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"locality": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"location_state": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"modified": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"organization": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"organizational_unit": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"public_key": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"remote_ip": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"serial_number": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"sha1_fingerprint": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"sha256_fingerprint": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"sha384_fingerprint": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"signature_algorithm": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"state": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"status": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"surname": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"type": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"uri": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"valid_from": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"valid_until": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"version": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+
+			"certificate_status": {
+				Type:     schema.TypeSet,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"chain_status": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"self_signed": {
+							Type:     schema.TypeBool,
+							Optional: true,
+						},
+						"trusted": {
+							Type:     schema.TypeBool,
+							Optional: true,
+						},
+					},
+				},
+			},
+			"created": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"etag": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"modified": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"name": {
```

Comment:
Make this Computed. Attributes  alias_name and remote_ip are used to make read call here, so except these values make rest of the values as Computed.